Dockerfile

```diff
@@ -62,6 +63,6 @@ COPY . /updater2
 RUN set -e; for D in . quazip fluid; do cd /updater2/$D && git clean -dXff; done
 WORKDIR /build
 RUN /qt/bin/qmake -config release /updater2 && make -j`nproc`
-RUN mv updater2 updater2-nonstripped
-RUN strip updater2-nonstripped -o updater2
-CMD cp updater2 updater2-nonstripped /build-docker
+RUN mv updater2 updater2-nonstripped && strip updater2-nonstripped -o updater2
+RUN ln -s updater2 unvanquished-updater && zip UnvUpdaterLinux.zip unvanquished-updater
```

Comment:
Why not just `Unvanquished Updater` or `UnvanquishedUpdater`? (Also, maybe we can use spaces in 2020).

Reply:
I thought this is the standard naming style for Linux binaries.

Reply:
changed to `UnvanquishedUpdater`